config/routes.rb

```diff
@@ -71,6 +70,7 @@
   resources :system_settings
   resources :teams
   resources :users
+  resources :glossary, only: [:index]
```

Comment:
An adjustment i'd suggest is to make it a [singular resource](https://edgeguides.rubyonrails.org/routing.html#singular-resources) instead. Not much would change in terms of code, but i think that feels more semantically precise.